### PR TITLE
added travis and fixed a bug

### PR DESCRIPTION
**Pull request by sih4sing5hong5**
#7 
Must add this repo before merge
https://travis-ci.org/
https://coveralls.io/repos/new

I reformat 2 pep8 syntax in another branch. #8

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+sudo: false
+language: python
+python:
+- '3.4'
+- '3.5'
+install:
+- pip install pymodbus3 pyserial
```
Comment (netrounds-rimek) [inline .travis.yml:7]: As you commented pymodbus3 is only for python3. There is a `py3` branch with python3 compatibile version, while `master` works only with python2. 
So change this PR to be merged towards `py3`, not `master` and in some free time I will make it work with both python versions

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+sudo: false
+language: python
+python:
+- '3.4'
+- '3.5'
+install:
```
Comment (netrounds-rimek) [inline .travis.yml:6]: from https://docs.travis-ci.com/user/languages/python/:
```
# command to install dependencies
install: "pip install -r requirements.txt"
```

File: test/testFatek.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+from unittest.case import TestCase
+
+from fatek.fatek import Fatek
+
+
+class testFatek(TestCase):
```
Comment (netrounds-rimek) [inline test/testFatek.py]: testFatek -> TestFatek

Comment (netrounds-rimek) [reply test/testFatek.py]: and rename that file to `test_fatek.py` in directory `tests`

**Review by netrounds-rimek — CHANGES_REQUESTED** [review]

File: test/testFatek.py (hunk 3 above)
Comment (sih4sing5hong5) [reply test/testFatek.py]: done

File: .travis.yml (hunk 1 above)
Comment (sih4sing5hong5) [reply .travis.yml:7]: I used `pip install pymodbus pyserial || pip install pymodbus3 pyserial` in #9.
It is more stable than branch version for the test.

**Pull request closed without merging**